pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -1497,6 +1498,48 @@ func TestGameServerDisableServiceAccount(t *testing.T) {
 	assert.Equal(t, "/var/run/secrets/kubernetes.io/serviceaccount", pod.Spec.Containers[0].VolumeMounts[0].MountPath)
 }
 
+func TestGameServerPassthroughPortAnnotation(t *testing.T) {
+	t.Parallel()
+	runtime.FeatureTestMutex.Lock()
+	defer runtime.FeatureTestMutex.Unlock()
+	require.NoError(t, runtime.ParseFeatures(string(runtime.FeatureAutopilotPassthroughPort)+"=true"))
+	gs := &GameServer{ObjectMeta: metav1.ObjectMeta{Name: "gameserver", UID: "1234"}, Spec: GameServerSpec{
+		Ports: []GameServerPort{
+			{Name: "defaultPassthrough", PortPolicy: Passthrough},
+			{Name: "defaultDynamic", PortPolicy: Dynamic, ContainerPort: 7654},
+			{Name: "defaultDynamicTwo", PortPolicy: Dynamic, ContainerPort: 7659},
+			{Name: "defaultPassthroughTwo", PortPolicy: Passthrough},
+		},
+		Template: corev1.PodTemplateSpec{
+			Spec: corev1.PodSpec{
```

Comment:
Please test the multi-container case. The `Ports` array can take a `container` name as well, so `Ports` could be a mix of different container ports.

Reply:
Added!